cmd/skaffold/app/cmd/util_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Skaffold Authors
+Copyright 2019 The Skaffold Authors
```

Comment:
```suggestion
Copyright 2020 The Skaffold Authors
```

Reply:
This change was made based on the review feedback form https://github.com/GoogleContainerTools/skaffold/pull/5117#discussion_r539492978

Reply:
ah, alright